### PR TITLE
Tag Juno.jl 0.2.0 and Atom.jl 0.5.2

### PR DESCRIPTION
**Pull request by pfitzseb**
Some bugfixes.

### CODE REVIEW

File: Juno/versions/0.2.0/requires [hunk 1]
```diff
@@ -0,0 +1,4 @@
+julia 0.5-
```
Comment (tkelman) [inline Juno/versions/0.2.0/requires]: Should Juno 0.1.0 be 0.5-only? It fails tests on 0.4.

Comment (pfitzseb) [inline Juno/versions/0.2.0/requires]: I think so, yes (cc @MikeInnes though). There should be no way Juno could get automatically installed on a 0.4 setup, so I'm not sure if it's worth changing.

**Comment by tkelman** [comment]
does it serve any purpose to allow it to be installed on 0.4 at all?

**Comment by tkelman** [comment]
does it serve any purpose to allow it to be installed on 0.4 at all?

**Comment by pfitzseb** [comment]
Don't think so -- I'd say it was wrong to have the 0.4 requirement in 0.1.0.

**Comment by tkelman** [comment]
which isn't this 0.1.0, but could be an 0.1.1.

**Pull request merged**